samples/drivers/adc/boards/lpcxpresso55s36.overlay

```diff
@@ -0,0 +1,33 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2022 NXP
+ */
+
+#include <zephyr/dt-bindings/adc/adc.h>
+
+/ {
+	zephyr,user {
+		/* adjust channel number according to pinmux in board.dts */
+		io-channels = <&adc0 0>;
+	};
+};
+
+&adc0 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	/*
+	 * This sample requires PIO1_9 found on J7 pin 1 as the ADC0 channel 3 input
+	 * be connected to various voltages to observe the change in reading.
+	 */
+
+	channel@0 {
+		reg = <0>;
+		zephyr,gain = "ADC_GAIN_1";
+		zephyr,reference = "ADC_REF_EXTERNAL0";
+		zephyr,acquisition-time = <ADC_ACQ_TIME_DEFAULT>;
+		zephyr,vref-mv = <3300>;
+		zephyr,resolution = <12>;
+	};
```

Comment:
there needs to be a `zephyr,input-positive` property here to meet the requirements of the new [LPADC driver rewrite](https://github.com/zephyrproject-rtos/zephyr/commit/e587047dc6b77c28ea399e272c41beb2cd52f142)